commons/src/main/java/org/infinispan/multimap/api/BasicMultimapCache.java

```diff
@@ -0,0 +1,184 @@
+package org.infinispan.multimap.api;
+
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+
+import org.infinispan.commons.util.Experimental;
+
+/**
+ * {@link BasicMultimapCache} provides the common building block for the two different types of multimap caches that Infinispan
+ * provides: embedded and remote. <p> Please see the <a href="http://infinispan.org/documentation/">Infinispan
+ * documentation</a> and/or the <a href="http://infinispan.org/docs/dev/getting_started/getting_started.html">5 Minute
```

Comment:
Are those tutorials relevant?

Reply:
This are already merged on master in the multimap embedded doc